.travis.yml

```diff
@@ -1,162 +0,0 @@
-      os: linux
-      rust: nightly
-      env: CI_STAGE_BUILD_RELEASE=yes
-
-    - name: Godot test (nightly)
-      os: linux
-      rust: nightly
-      env: CI_STAGE_GODOT_TEST=yes RUST_BACKTRACE=1 GODOT_VER=3.2 GODOT_REL=stable
-      before_script:
-        - 'wget "https://downloads.tuxfamily.org/godotengine/$GODOT_VER/Godot_v${GODOT_VER}-${GODOT_REL}_linux_headless.64.zip" -O /tmp/godot.zip'
-        - unzip /tmp/godot.zip -d godot_bin
-        - export PATH=$PATH:$PWD/godot_bin/
-
-    - name: cargo test (stable, windows)
-      os: windows
-      rust: stable
-      env: CI_STAGE_CARGO_TEST=yes CI_MSVC_TEST=yes
-      before_script:
-        - rustup target add x86_64-pc-windows-msvc
-
-    - name: cargo build --release (stable, windows)
-      os: windows
-      rust: stable
-      env: CI_STAGE_BUILD_RELEASE=yes CI_MSVC_TEST=yes
-      before_script:
-        - rustup target add x86_64-pc-windows-msvc
-
-    - name: cargo test (nightly, windows)
-      os: windows
-      rust: nightly
-      env: CI_STAGE_CARGO_TEST=yes CI_MSVC_TEST=yes
-      before_script:
-        - rustup target add x86_64-pc-windows-msvc
-
-    - name: cargo build --release (nightly, windows)
-      os: windows
-      rust: nightly
-      env: CI_STAGE_BUILD_RELEASE=yes CI_MSVC_TEST=yes
-      before_script:
-        - rustup target add x86_64-pc-windows-msvc
-
-    - name: cargo test ios-simulator
-      os: osx
-      osx_image: xcode11.4
-      rust: stable
-      env: CI_STAGE_IOS_SIMULATOR_TEST=yes
-      before_script:
-       - rustup target add x86_64-apple-ios
-       # download cargo-dinghy and set it up for running on an emulator.
-       - curl -L https://github.com/snipsco/dinghy/releases/download/cargo-dinghy%2F0.4.37/cargo-dinghy-macos.tgz -o cargo-dinghy-macos.tar.gz
-       - tar -zxvf cargo-dinghy-macos.tar.gz
-       - mkdir -p $HOME/.cargo/bin
-       - cp cargo-dinghy-macos/cargo-dinghy $HOME/.cargo/bin
-
-script:
-  - if [[ "$CI_STAGE_CARGO_FMT" == "yes" ]]; then
-      cargo fmt --all -- --check;
-    fi
-
-  - if [[ "$CI_STAGE_IOS_SIMULATOR_TEST" == "yes" ]]; then
-      RUNTIME_ID=$(xcrun simctl list runtimes | grep iOS | cut -d ' ' -f 7 | tail -1);
-      export SIM_ID=$(xcrun simctl create My-iphone11 com.apple.CoreSimulator.SimDeviceType.iPhone-11 $RUNTIME_ID);
-      xcrun simctl boot $SIM_ID;
-      cd gdnative-core;
-      cargo dinghy --platform auto-ios-x86_64 test;
-      cd ..;
-      cd gdnative-sys;
-      cargo dinghy --platform auto-ios-x86_64 test;
-    fi
-
-  - if [[ "$CI_STAGE_CARGO_CLIPPY" == "yes" ]]; then
-      cargo clippy --all --all-features -- -D clippy::style -D clippy::complexity -D clippy::perf -D clippy::dbg_macro -D clippy::todo -D clippy::unimplemented;
-    fi
-
-  - if [[ "$CI_STAGE_CARGO_TEST" == "yes" ]]; then
-      cargo test --all --all-features;
-      if [[ "$CI_MSVC_TEST" == "yes" ]]; then
-        cargo test --target x86_64-pc-windows-msvc --all --all-features;
-      fi
```

Comment:
This was not migrated either. 

I'm not sure what the difference is between the first and second targets. Is it that the second one is 64 bit and the first is 32?

Reply:
The MSVC ABI is different from the GNU one (used by default on Travis, was it mingw or cygwin?), and this has caused problems with platform-dependent code previously. I'll add it back.